client/src/containers/SignUp/SignUp.js

```diff
@@ -1,23 +1,23 @@
 import React from 'react'
-import { AuthForm } from '../../components'
+import { Form } from '../../components'
 import { handleSubmit } from './utils';
-import { schema, inputs} from './Schema';
+import { schema, inputs } from './Schema';
 
 const SignUp = (props) => {
-    return (
-        <div>
-          <AuthForm
-            {...props} 
-            handleSubmit={handleSubmit}
-            schema={schema}
-            inputs={inputs}
-            submitLabel="SignUp"
-            footerLabel="Already a registered user?"
-            linkTo="/signin"
-            linkToLabel="SignIn"
-          />
-        </div>
-    )
+  return (
+    <div>
+      <Form
+        {...props}
+        handleSubmit={handleSubmit}
+        schema={schema}
+        inputs={inputs}
+        submitLabel="SignUp"
+        footerLabel="Already a registered user?"
+        linkTo="/signin"
```

Comment:
You can create a object because both are related to link to, instead of multiple props.